funding/commitment_type_negotiation.go

```diff
@@ -354,15 +368,17 @@ func implicitNegotiateCommitmentType(local,
 }
 
 // hasFeatures determines whether a set of features is supported by both the set
-// of local and remote features.
+// of local and remote features. The set of missing feature bits is also
+// returned.
 func hasFeatures(local, remote *lnwire.FeatureVector,
-	features ...lnwire.FeatureBit) bool {
+	features ...lnwire.FeatureBit) (bool, []lnwire.FeatureBit) {
 
+	var missing []lnwire.FeatureBit
 	for _, feature := range features {
 		if !local.HasFeature(feature) || !remote.HasFeature(feature) {
-			log.Infof("feature %d missing in local %t or in remote %t", feature, local.HasFeature(feature), remote.HasFeature(feature))
-			return false
```

Comment:
this was added in previous commit... so the previous commit is then no longer needed

Reply:
ups!